jdaviz/configs/specviz/helper.py

```diff
@@ -271,7 +271,7 @@ def set_spectrum_tick_format(self, fmt, axis=None):
             self._default_spectrum_viewer_reference_name
         ).figure.axes[axis].tick_format = fmt
 
-    def get_data(self, data_label=None, cls=None, subset_to_apply=None):
+    def get_data(self, data_label=None, spectral_subset=None, cls=None, **kwargs):
```

Comment:
having `**kwargs` here scares me a bit, unless we make sure that nothing ineligible is passed (otherwise a typo could easily get swallowed without an error, if a user type `get_data(spctral_subset='Subset 1')` that will get sent to `**kwargs` and `spectral_subset` remains as `None`)

Reply:
Added a check so that it could only contain those two parameters with an error message that should be helpful.